tests/integration_tests/test_kani.py

```diff
@@ -19,15 +21,23 @@ def test_kani(results_dir):
     """
     Test all Kani proof harnesses.
     """
+    # https://psutil.readthedocs.io/en/latest/#psutil.virtual_memory confusingly enough returns
+    # information about the physical memory. Limiting to 64GB should exclude most consumer hardware,
+    # preventing the test from locking up peoples computers, while still allowing them to force their
+    # way through it with enough swap.
+    if (
+        psutil.swap_memory().total + psutil.virtual_memory().total
+        <= 64 * 1000 * 1000 * 1000
```

Comment:
```suggestion
        <= 64 * 1024 * 1024 * 1024
```

Reply:
Ah, whoops, I do mean `GB` here, forgot update the skip message

Reply:
Its unusual to specifically use GB instead of GiB why would we do it in this case?

Reply:
I agree with Jonathan here. I prefer GiB when it comes to memory.